Bug: CI fails on docker-compose

### Which package(s) does this bug affect?

- [X] Create Plugin
- [ ] Sign Plugin
- [ ] Plugin E2E
- [ ] Plugin Meta Extractor

### Package versions

```shell
5.2.1
```


### What happened?

The CI workflow fails on docker-compose.

```
Run docker-compose pull
/home/runner/work/_temp/4ba78501-a7a6-4b1f-87c0-79132bbc0932.sh: line 1: docker-compose: command not found
Error: Process completed with exit code 127.
```

Here is a possible solution:

<img width="616" alt="Screenshot 2024-08-08 at 9 41 20 AM" src="https://github.com/user-attachments/assets/a5ba68b4-8e4f-4be5-9b32-1b9e7c896da2">

See error here:

https://github.com/nicfv/Psychart/actions/runs/10170100986/job/28524463536#step:8:6

### What you expected to happen

The CI workflow to pass.

### How to reproduce it (as minimally and precisely as possible)

1. `npx @grafana/create-plugin`
2. Push to repository
3. Check CI workflow

### Environment

```shell
System:
    OS: macOS 14.5
    CPU: (8) arm64 Apple M2
    Memory: 436.59 MB / 24.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.11.1 - ~/.nvm/versions/node/v20.11.1/bin/node
    npm: 10.8.2 - ~/.nvm/versions/node/v20.11.1/bin/npm
  Browsers:
    Brave Browser: 127.1.68.137
    Chrome: 105.0.5195.102
    Safari: 17.5
```


### Additional context

_No response_

--- a/packages/create-plugin/templates/github/workflows/ci.yml
+++ b/packages/create-plugin/templates/github/workflows/ci.yml
@@ -169,8 +169,8 @@ jobs:
 
       - name: Start Grafana
         run: |
-          docker-compose pull
-          DEVELOPMENT=false GRAFANA_VERSION=$\{{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=$\{{ matrix.GRAFANA_IMAGE.NAME }} docker-compose up -d
+          docker compose pull
+          DEVELOPMENT=false GRAFANA_VERSION=$\{{ matrix.GRAFANA_IMAGE.VERSION }} GRAFANA_IMAGE=$\{{ matrix.GRAFANA_IMAGE.NAME }} docker compose up -d
 
       - name: Wait for Grafana to start
         uses: nev7n/wait_for_response@v1
@@ -193,7 +193,7 @@ jobs:
           docker logs {{ pluginId }} >& grafana-server.log
 
       - name: Stop grafana docker
-        run: docker-compose down
+        run: docker compose down
 
       - name: Upload server log
         uses: actions/upload-artifact@v4

